lte/protos/session_manager.proto

```diff
@@ -52,6 +52,8 @@ message CommonSessionContext {
   string apn = 4;
   bytes msisdn = 5;
   RATType rat_type = 6;
+  string ue_ipv6 = 7;
+  uint32 pdn_type = 8;
```

Comment:
do we need to set the pdn type here? Sessiond and session_proxy will check both ue_ipv4 and ue_ipv6 so we will know the type checking if the value exists

Reply:
Can a session have both, ipv4 and ipv6 simultaneously?

Reply:
Instead of deriving pdn type from the ip addresses received, it would be better to explicitly send the pdn type. If the user requested for ipv4v6 and for some reason one of the ip addresses say ipv4 is corrupted and is sent as NULL to session manager, then session manager might wrongly derive the pdn type as ipv6 instead of ipv4v6. If pdn type is present, in this case session manager can establish an ipv6 session and return an error cause

Reply:
A session can have ipv4 and v6 simultaneously